### PR TITLE
rewrite intro and abstract

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

**Review by EskoDijk — CHANGES_REQUESTED** [review]
Overall the new text isn't clear enough yet. In the current proposal we see a mix of paragraphs that have a different viewpoint on what this document is:
1) "old" viewpoint - this document just describes a constrained version of the voucher and nothing more. E.g. the title also reflects this. Bootstrap protocol use of HTTP or CoAP should be irrelevant to this. We don't define a complete constrained BRSKI solution but only a Voucher format that other docs can use.
2) "new" viewpoint - we define constrained BRSKI including all it needs like voucher format, protocol use of CoAP, CoAP resources, Registrar-to-MASA communication, etc.  A much wider scope!

Before I can propose sensible review comments we must first decide what our aim is with this document.  Should we make a split between constrained voucher format and "rest" of the bootstrap protocol, like BRSKI did? Or do we want to integrate everything into one draft?

**Comment by petervanderstok** [comment]
Hi Esko,

Good point.
I like a constrained "pun" purpose document.
However, some assumptions on the protocol should be described.
I would prefer not to go into the options of how to use the voucher
contents and under which circumstances.
For me that is clearly another document.
The motivation why the different items are in the voucher or VR should
be discussed wen they exceed the brski document.

Peter
Esko Dijk schreef op 2021-01-07 10:37:

> @EskoDijk requested changes on this pull request. 
> 
> Overall the new text isn't clear enough yet. In the current proposal we see a mix of paragraphs that have a different viewpoint on what this document is: 
> 
> * "old" viewpoint - this document just describes a constrained version of the voucher and nothing more. E.g. the title also reflects this. Bootstrap protocol use of HTTP or CoAP should be irrelevant to this. We don't define a complete constrained BRSKI solution but only a Voucher format that other docs can use.
> * "new" viewpoint - we define constrained BRSKI including all it needs like voucher format, protocol use of CoAP, CoAP resources, Registrar-to-MASA communication, etc. A much wider scope!
> 
> Before I can propose sensible review comments we must first decide what our aim is with this document. Should we make a split between constrained voucher format and "rest" of the bootstrap protocol, like BRSKI did? Or do we want to integrate everything into one draft? 
> 
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
 

Links:
------
[1]
https://github.com/anima-wg/constrained-voucher/pull/68#pullrequestreview-563338595
[2]

**Review by EskoDijk — APPROVED** [review]

**Comment by EskoDijk** [comment]
Proposed further edits in my last commit, for review!

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -96,49 +96,45 @@ informative:
 
 --- abstract
 
-This document defines a strategy to securely assign a pledge to an
-owner, using an artifact signed, directly or indirectly, by the
-pledge's manufacturer.  This artifact is known as a "voucher".
+This document defines a protocol to securely assign a pledge to an
+owner and to enroll it into the owner's network. The protocol uses an artifact that is signed, directly 
+or indirectly, by the pledge's manufacturer.  This artifact is known as a "voucher".
```
Comment (mcr) [inline constrained-voucher.mkd:101]: What is the "indirect" situation?

Comment (mcr) [inline constrained-voucher.mkd:101]:
```suggestion
This document defines a protocol to securely assign a pledge to an
owner and to enroll it into the owner's network. 
The protocol uses an artifact that is signed by the pledge's manufacturer.  
This artifact is known as a "voucher".
```

**Pull request merged**